bots/hermes/makefile

```diff
@@ -0,0 +1,42 @@
+.DEFAULT_GOAL=help
+
+VENVPATH=venv
+PYTHON=$(VENVPATH)/bin/python3
+
+venv: $(VENVPATH)/bin/activate
+$(VENVPATH)/bin/activate: requirements.txt
+	test -d $(VENVPATH) || virtualenv -p python3 $(VENVPATH); \
+	. $(VENVPATH)/bin/activate; \
+	pip install -r requirements.txt; \
+	touch $(VENVPATH)/bin/activate;
+
+
+##venv: setup your dev environment
+install-deps: venv
+
+##run: run the indexer
+## 		example: make run
+run: install-deps
+	GOOGLE_APPLICATION_CREDENTIALS=.secrets/service-account.json $(PYTHON)
```

Comment:
I guess you haven't specified which script to run with your `$(PYTHON)` binary.
I recommend you to test your makefile rules before you submit them in a pull request